common/baselines/librispeech/ls960/gmm/baseline_args.py

```diff
@@ -0,0 +1,457 @@
+    :param hmm_states: number of states per label for the HMM-topology
+    :param feature_flow:
+    :param add_unknown: set to true if an unknown phoneme exists
+    """
+
+    CartQuestions = (
+        CartQuestionsWithStress if use_stress_marker else CartQuestionsWithoutStress
+    )
+
+    cart_questions_class = CartQuestions(
+        max_leaves=max_leaves,
+        min_obs=min_obs,
+        add_unknown=add_unknown,
+    )
+
+    cart_questions = cart.PythonCartQuestions(
+        phonemes=cart_questions_class.phonemes_boundary_special,
+        steps=cart_questions_class.steps,
+        max_leaves=max_leaves,
+        hmm_states=hmm_states,
+    )
+
+    cart_lda_args = {
+        "name": "cart_mono",
+        "alignment": "train_mono",
+        "initial_flow_key": feature_flow,
+        "context_flow_key": feature_flow.split("+")[0],
+        "context_size": 9,
+        "num_dim": 48,
+        "num_iter": 2,
+        "eigenvalue_args": {},
+        "generalized_eigenvalue_args": {"all": {"verification_tolerance": 1e16}},
+    }
+
+    return util.GmmCartArgs(
+        cart_questions=cart_questions,
+        cart_lda_args=cart_lda_args,
+    )
+
+
+def get_triphone_args(feature_flow: str = "mfcc+context+lda") -> util.GmmTriphoneArgs:
+    """
+    :param feature_flow:
+    """
+
+    triphone_training_args = {
+        "name": "tri",
+        "initial_alignment": "train_mono",
+        "feature_flow": feature_flow,
+        "splits": 10,
+        "accs_per_split": 2,
+        "align_extra_rqmt": {"mem": 4},
+        "accumulate_extra_rqmt": {"mem": 4},
+        "split_extra_rqmt": {"mem": 4},
+    }
+
+    triphone_recognition_args = {
+        "iters": [8, 10],
+        "feature_flow": feature_flow,
+        "pronunciation_scales": [1.0],
+        "lm_scales": [25],
+        "lm_lookahead": True,
+        "lookahead_options": None,
+        "create_lattice": True,
+        "eval_single_best": True,
+        "eval_best_in_lattice": True,
+        "search_parameters": {
+            "beam_pruning": 15.0,
+            "beam-pruning-limit": 100000,
+            "word-end-pruning": 0.5,
+            "word-end-pruning-limit": 10000,
+        },
+        "lattice_to_ctm_kwargs": {
+            "fill_empty_segments": False,
+            "best_path_algo": "bellman-ford",
+        },
+        "optimize_am_lm_scale": True,
+        "rtf": 20,
+        "mem": 4,
+        "parallelize_conversion": True,
+    }
+
+    sdm_args = {
+        "name": "sdm.tri",
+        "alignment": "train_tri",
+        "feature_flow_key": feature_flow,
+    }
+
+    return util.GmmTriphoneArgs(
+        training_args=triphone_training_args,
+        recognition_args=triphone_recognition_args,
+        sdm_args=sdm_args,
+    )
+
+
+def get_vtln_args(feature_flow: str = "mfcc+context+lda") -> util.GmmVtlnArgs:
+    """
+    :param feature_flow:
+    """
+    vtln_training_args = {
+        "feature_flow": {
+            "name": f"uncached_ {feature_flow}",
```

Comment:
This space is probably wrong?
```suggestion
            "name": f"uncached_{feature_flow}",
```